build documentation locally with make html on mac fails

## make html on mac fails

Page URL: https://github.com/qgis/QGIS-Documentation


I work on a mac running MacOS Montery version 12.7.6
I followed instructions in Readme of https://github.com/qgis/QGIS-Documentation how to build documentation locally:

When I run make html I get following error!

**(venv) MacBook-Pro-van-Diethard:QGIS-Documentation diethard$ make html**
```
echo "  -D language=en -t i18n"
  -D language=en -t i18n
if [ en != "en" ]; then \
		sphinx-build -b html "." "build/html/en"  -D language=en -t i18n ; \
	else \
		sphinx-build -b html -nW --keep-going "." "build/html/en"  ; \
	fi
Traceback (most recent call last):
  File "/Users/diethard/Documents/ontwikkeling/QGIS-Documentation/venv/bin/sphinx-build", line 8, in <module>
    sys.exit(main())
  File "/Users/diethard/Documents/ontwikkeling/QGIS-Documentation/venv/lib/python3.10/site-packages/sphinx/cmd/build.py", line 369, in main
    locale.setlocale(locale.LC_ALL, '')
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/locale.py", line 620, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
make: *** [html] Error 1

```
After changing locally the following statement in function main in python file build.py
from 

```
def main(argv: Sequence[str] = (), /) -> int:
    locale.setlocale(locale.LC_ALL, '')

```
to
```
def main(argv: Sequence[str] = (), /) -> int:
    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
```

And I save and run make html again.
The documentation is build.

```
build succeeded.
The HTML pages are in build/html/en.
```

The build also seems to work ok when I provide language parameter

**(venv) MacBook-Pro-van-Diethard:QGIS-Documentation diethard$ make LANG=nl html**

```
build succeeded.
The HTML pages are in build/html/nl.
```

I looked up the result and saw several translated images in the build. 
(I have not downloaded translations from transifex, to see if everything is correctly translated as well)

So I can continue now, but without changing the _requirements_ I wonder if other mac users trying to build documentation run into the same problem.

--- a/README.md
+++ b/README.md
@@ -94,15 +94,12 @@ locale.Error: unsupported locale setting
 make: *** [html] Error 1
 ```
 
-A solution is to open the python file build.py inside venv folder with the internal pico editor.
-Use in path your installed python version, I installed python 3.10.
+A solution is to edit the Python file `build.py` inside your `venv` folder with a text editor (pico, textEdit, vscode,...).
+The file is stored at `./venv/lib/python3.10/site-packages/sphinx/cmd/build.py`
+(replace `python3.10` with your installed Python version).
 
-```sh
-pico ./venv/lib/python3.10/site-packages/sphinx/cmd/build.py
-```
-
-Goto function main, in pico use **Ctrl-W** and use: 'def main' as search string.
-Change the 1st line in function main from:
+1. Open the file in the text editor
+2. Search and replace:
 
 ```
 def main(argv: Sequence[str] = (), /) -> int:

